src/pudl/transform/ferc1.py

```diff
@@ -318,6 +319,7 @@ def align_row_numbers_dbf(
             )
 
         df = pd.merge(df, row_map, on=["report_year", "row_number"], how="left")
+        df.to_csv("aligned.csv")
```

Comment:
Are these `df.to_csv()` calls debugging outputs you meant to remove?

You can also use the cached dataframes that are optionally stored within the TableTransformer for this kind of debugging.

Reply:
WHoops these were just for some quick and dirty debugging. I was just being lazy and decided to add these in for a quick check instead of using the cached dataframes